### ISSUE TITLE
More water

### ISSUE THREAD

**Issue by Extex101** [minetest/minetest_game#2249]
Minetest Needs more water maybe an ocean biome

The comparison between MC map and MT map 

### MC MAP
![maxresdefault-1](https://user-images.githubusercontent.com/33675971/47322201-01073000-d60c-11e8-858b-b06abdeaa221.jpg)

MT MAP
![2018-10-22 14 54 29](https://user-images.githubusercontent.com/33675971/47322241-23994900-d60c-11e8-8104-45e902efca31.png)
As you can see Minetest's map looks 
### VASTLY BETTER

But you may notice that Minetest has much less water

Basically add more water to minetest

**Comment by NathanSalapat** [comment]
What advantage would there be to having more water? Currently Minetest_Game doesn't have much in the way of interest in the water.

**Comment by tenplus1** [comment]
It all depends what biomes are added as to how much water is shown, try using the valleys mapgen since it offers rivers and gives more water.

**Comment by paramat** [comment]
Engine issue.
Your map is of mgv7.
The ratio of land to water is determined by the core mapgen default parameters. These won't be changed for the stable mapgens mgv6 or mgv7 but the newer unstable mapgens (mgvalleys, mgcarpathian) could be tuned.
Try mapping one of the newer mapgens to see what the ratio is.
///////////////////

Related, i have been experimenting with a new core mapgen that creates continents, this is roughly what a whole world (62000x62000) looks like (terrain incomplete and WIP):

![mgw3s31h](https://user-images.githubusercontent.com/3686677/47342097-bb457880-d69a-11e8-8f49-528d33c9f9fc.png)

**Comment by Fixer-007** [comment]
You can increase amount of water via mapgen parameters, but don't forget to alter biome settings (?)

**Comment by paramat** [comment]
It's easier to do this by moving terrain downwards, not by raising 'water_level', because the entire biome system is defined around 'water_level' being y = 1, so you would need to alter all the y-limits for every biome. Either way it's not a simple thing to do.

**Comment by paramat** [comment]
Possible, but codewise it's much simpler and much more preferable to shift terrain up and down instead.

At one point a while back i added a way to shift the 'zero point' of biomes per mapchunk, in order to have the registered biomes repeat in the mgv7 floatlands. I reverted that as i soon realised how it made so much other code a nightmare. Afterwards as i worked on other new features i realised further what a good idea it was to revert that :)
